.eslintrc

```diff
@@ -10,6 +10,8 @@
     "curly": ["error", "multi-line"],
     "brace-style": ["error", "stroustrup"],
     "react/prop-types": 0,
+    "array-callback-return": 0,
```

Comment:
Why did you disable this rule?

Reply:
The linter was pestering me, even after I changed it!